### PR TITLE
Help cleans resnet train.ipynb -- use relative path imports, rid ACT module

### PR DESCRIPTION
**Pull request by peteflorence**
Hey there, thanks for this great repo.  Very easy to use and clean implementation.

This tiny PR cleans a couple things that might be helpful for others.  It brings in the nice relative path imports in other notebooks that weren't in `resnet_34_8s_train.ipynb`.  Also there was an import from `adaptive_computation_time`, but this module isn't needed.

If I get around to it later I might also help add some documentation on setting up VOC for use with this repo.

Cheers!

### CODE REVIEW

**Comment by peteflorence** [comment]
Also just tacked on a small fix to README

File: pytorch_segmentation_detection/recipes/pascal_voc/segmentation/resnet_34_8s_train.ipynb [hunk 1]
```diff
@@ -11,8 +11,8 @@
     "%matplotlib inline\n",
     "\n",
     "import sys, os\n",
-    "sys.path.append(\"/home/daniil/repos/pytorch-segmentation-detection/\")\n",
-    "sys.path.insert(0, '/home/daniil/repos/pytorch-segmentation-detection/vision/')\n",
+    "sys.path.insert(0, '../../../../vision/')\n",
```
Comment (warmspringwinds) [inline pytorch_segmentation_detection/recipes/pascal_voc/segmentation/resnet_34_8s_train.ipynb:14]: Would be great if you can just define a variable with path to the cloned github repository
which will be used in ```sys.path()```

**Comment by warmspringwinds** [comment]
Thank you, @peteflorence !

Have a look at my comment

**Comment by warmspringwinds** [comment]
@peteflorence  I have just realized that you did it much easier for other people to use.

Thank you

**Comment by peteflorence** [comment]
Very good!

I've used your repo as a building block in a new project, hopefully will open source that soon

**Comment by warmspringwinds** [comment]
@peteflorence I am glad this repo worked for you :)
Great :) Let me know if you do, I would love to have a look

**Comment by peteflorence** [comment]
Hey @warmspringwinds, circling back on this, we just had our project go live that built in part off this repo.

Here is [code](https://github.com/RobotLocomotion/pytorch-dense-correspondence), [video](https://www.youtube.com/watch?v=L5UW1VapKNE), [arXiv pre-print](https://arxiv.org/pdf/1806.08756.pdf).

Your repo was very helpful -- my first version of training up our networks used a derived version of your training notebook, just with new pre-processing and loss function swapped in (we are doing a different task than segmentation).

Cheers!, Pete

**Comment by warmspringwinds** [comment]
@peteflorence 
That's a great work! I will have a more thorough look these days for sure!

I am glad that my work was useful for you :+1:
